test/core/loader/container.test.js

```diff
@@ -0,0 +1,37 @@
+import { createMockGmi } from "../../mock/gmi.js";
+import { domElement } from "../../mock/dom-element.js";
+import { getContainerDiv } from "../../../src/core/loader/container.js";
+import { gmi } from "../../../src/core/gmi/gmi.js";
+
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
```

Comment:
does it not complain that this is not at the top? 😲

Reply:
Ha. Will update. Good spot!